Documentation/special-services/pod-mutator.md

```diff
@@ -0,0 +1,76 @@
+# PodMutator Service
+To provide pod mutation capabilities for finer-grained control over rkt containerization concepts and debugging needs, 
+this proposal introduces a new optional stage1 app/service with special semantics that is lightweight.
+
+When added to a pod manifest, the PodMutator service, which is just an app with special semantics, 
```

Comment:
It will be injected into a pod using the new cmd 'rkt app inject <pod-uid> <pod-mutator-aci>'